docker-compose.yml

```diff
@@ -56,16 +56,6 @@ services:
     image: wal-g/pg
     container_name: wal-g_pg
 
-  pg_tests:
```

Comment:
We use it for local testing.

Reply:
Oh yeah this was super confusing. I can add it back. Any other issues/comments?

Reply:
Looks pretty good for me. Just add `pg_tests` back and I'll merge it.

Reply:
@dyson ping :)

Reply:
Thanks @g0djan will do this now!

Reply:
Added the pg_tests back and rebased on master as there we some overlap with merged code since I branched. Should be good to merge now.